pom.xml

```diff
@@ -55,6 +61,14 @@
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<configuration>
+					<source>9</source>
+					<target>9</target>
+				</configuration>
+			</plugin>
```

Comment:
This plugin is already defined in spring boot parent pom. I think you just need to change `<java.version>1.8</java.version>`  in line no.18 to change the java version

Reply:
yeah, I changed the version now